CONTRIBUTING.md

```diff
@@ -6,11 +6,15 @@ Welcome and thanks for your interest in contributing to our project. 🙇‍♀
 
 The tag "good first issue" and "help wanted" highlights some tickets that we'd love help with. Feel free to drop us a note if you'd like to work on these, especially if you'd like to tackle a more complex issue where it might be helpful to discuss approach. For privacy reasons, we don't allow Airtable access to the general public, but we're happy to answer questions about the table structure for contributors.
 
-Please check with us before working on tickets in the "Icebox / Ideas" column or with the "backburner" label.
+Please check with us before working on tickets in the "Icebox / Ideas" column or with the "backburner" label. Please also check before working on a ticket already assigned to somebody.
+
+## Using git
+
+We prefer merges instead of rebasing. Merges to master will be squashed. Please don't use `force-push` as it messes with Pull request comments and makes it hard for others to look at your branch.
```

Comment:
Oh, I didn't know about this preference, I was using rebase myself most of the time lol I'll stick to merge then 🙇

Reply:
We could change it if you disagree. I've set github to auto-squash changes when merging to master via the button, and force-push created a lot of headaches at my last job so I really encourage people to find a different way.